dashboards/TDinsight.md

```diff
@@ -195,9 +195,15 @@ For special use cases, `-O` would set the organization id when you use Grafana C
 Install the TDengine data-source plugin from GitHub.
```

Comment:
For aligning with Chinese version, how about change to following:
Install the latest version of TDengine data-source plugin from GitHub.

Reply:
Changed.